CollAction/Views/Account/Login.cshtml

```diff
@@ -15,6 +15,10 @@
     var canTrack = consentFeature.CanTrack;
 }
 
+@section styles {
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css">
```

Comment:
Could you keep the integrity attribute? As discussed in the call, that does help.

Reply:
fixed